### DISCUSSION TITLE
KMemo: New page

### DISCUSSION THREAD

**Discussion by radekjindra** [kryslt/KControls#58] [Q&A] [answered]
Is there ALt+Enter [new page] and header and footer in Kmemo? 
Thank you for answer

**Comment by kryslt** [comment, marked as answer]
New page break: No, sections are not supported yet. Things like Ctrl+Enter in MS Word will not work.
Header + Footer: No customizable headers and footers are supported yet. You can only use the default header and footer as implemented in the TKPrintPageSetup component. It can print the document title in the page header and page numbers in the page footer.

**Reply by radekjindra** [reply]
Thank you
